escuela-nacional-de-antropologia-e-historia-author-date.csl

```diff
@@ -489,7 +489,7 @@
     </layout>
   </citation>
   <!-- Para generar la bibliografía -->
-  <bibliography et-al-min="4" et-al-use-first="3" subsequent-author-substitute="" entry-spacing="0">
+  <bibliography et-al-min="4" et-al-use-first="3" subsequent-author-substitute="" entry-spacing="-3">
```

Comment:
The CSL specification explicitly forbids negative numbers for "entry-spacing", and it probably will give problems with some CSL-based software.